src/pages/Graph/SummaryPanelGroup.tsx

```diff
@@ -239,21 +238,13 @@ export default class SummaryPanelGroup extends React.Component<SummaryPanelPropT
       <>
         {hasCB && (
           <div>
-            <Icon
-              name={icons.istio.circuitBreaker.name}
-              type={icons.istio.circuitBreaker.type}
-              style={{ width: '10px' }}
-            />
+            <KialiIcon.CircuitBreaker />
             <span style={{ paddingLeft: '4px' }}>Has Circuit Breaker</span>
           </div>
         )}
         {hasVS && (
           <div>
-            <Icon
-              name={icons.istio.virtualService.name}
-              type={icons.istio.virtualService.type}
-              style={{ width: '10px' }}
-            />
+            <KialiIcon.VirtualService />
```

Comment:
Apparently, this is rendering the missing sidecar icon.
![image](https://user-images.githubusercontent.com/23639005/67592215-c6573c80-f724-11e9-8c32-3f828302f8fc.png)